cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Query.hs

```diff
@@ -244,6 +247,21 @@ hprop_cli_queries = integrationWorkspace "cli-queries" $ \tempAbsBasePath' -> H.
       pure ()
 
     )
+  where
+  patchGovStateOutput :: String -> Either String String
```

Comment:
Please do not use `String` as the error in an `Either`.

Reply:
Solved [here](https://github.com/IntersectMBO/cardano-node/commit/c6a953cfbbcbde21386d191be207f955e4dd4c1d)